### ISSUE TITLE
Clarify warning message around setting config options

### ISSUE THREAD

**Issue by dustymabe** [minishift/minishift#708]
```
[student@workstation ~]$ minishift config set memory 10240
To apply the changes, you must delete the current VM with `minishift delete` and start a new VM with `minishift start`.
[student@workstation ~]$ minishift config set cpus 4
To apply the changes, you must delete the current VM with `minishift delete` and start a new VM with `minishift start`.
```

I get these messages even if there is no "current VM". We should either not print these messages out if there is no current VM or we should re-word so that is doesn't imply that there is a current VM.

**Comment by hferentschik** [comment]
> I get these messages even if there is no "current VM". We should either not print these messages out if there is no current VM or we should re-word so that is doesn't imply that there is a current VM.

I think there is even another issue for that.

**Comment by LalatenduMohanty** [comment]
This is a duplicate of https://github.com/minishift/minishift/issues/471

**Comment by hferentschik** [comment]
Merged via pull request #731
